Lib/test/test_future.py

```diff
@@ -284,6 +284,7 @@ def test_annotations(self):
         eq("(x:=10)")
         eq("f'{(x:=10):=10}'")
 
+    def test_fstring_annotations(self):
```

Comment:
The above tests are also for f-string annotations. Actually the whole class is named AnnotationsFutureTestCase. I suggest to use the word "debug" in the name of this test (or what is the official name of this feature?).

Reply:
There's no official name. There's an open issue to pick a name and document it all, but I have no suggestions for it. I'll change the name to include "debug", since f-strings are already tested elsewhere.